project.mml

```diff
@@ -939,19 +908,14 @@ Layer:
           FROM ( -- subselect that contains both roads and rail/aero
             SELECT
                 way,
-                'highway_' || highway AS feature, --only motorway to tertiary links are accepted later on
-                horse,
-                foot,
-                bicycle,
+                'highway_' || (CASE WHEN highway = 'path' THEN carto_path_type(bicycle, horse) ELSE highway END) AS feature, --only motorway to tertiary links are accepted later on
```

Comment:
`highway = 'path'` will never be true here. Either remove the case, or change the comment to reflect this.

Reply:
Why will highway never be 'path' here?

Reply:
because the WHERE clause excludes it, as mentioned in the comment to the right

Reply:
I am not sure if we are looking at the same code - the WHERE condition for this query as i see it is here:

https://github.com/gravitystorm/openstreetmap-carto/blob/194d6419989240babd959a6b66754e82513d160d/project.mml#L932-L933

Reply:
The `only motorway to tertiary links are accepted later on` comment is rather cryptic. It seems more relevant for the surrounding `SELECT` where `_link` is stripped off.

It may be better to delete the comment as it doesn't obviously document the current code.

Reply:
The comment is remaining from an earlier version of the code (https://github.com/gravitystorm/openstreetmap-carto/commit/e4f1070dd9a28919a40b0d9f6bfb8177c752359d) where the '_link' was stripped of irrespective of the prefix and the comment indicated that this is all right because only the specific link types were ultimately rendered.  It should be removed.